x-pack/plugins/security_solution/public/common/components/callouts/callout_types.ts

```diff
@@ -5,7 +5,9 @@
  * 2.0.
  */
 
-export type CallOutType = 'primary' | 'success' | 'warning' | 'danger';
+import { EuiCallOutProps } from '@elastic/eui';
+
+export type CallOutType = NonNullable<EuiCallOutProps['color']>;
```

Comment:
NOTE: This removes the duplicates of the types from EuiCallOutProps types being used here but removes the `undefined` from it. If they add or remove one we get it for "free" and exhaustive types against it will signal type errors during the code review phase.